src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -12,6 +12,46 @@
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
     public final String feedbackToUser;
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Represents the result of a command execution.
+ */
+public class CommandResult {
+
+    /** The feedback message to be shown to the user. Contains a description of the execution result */
+    private final String feedbackToUser;
+
+    public String returnFeedbackToUser() {
```

Comment:
Good catch on creating the return function, but usually we name them as get<variable name>/ set<variable name>. Eclipse allows you to auto-generate these two functions for data members.